sdk/communication/communication-phone-numbers/test/public/utils/recordedClient.ts

```diff
@@ -63,6 +63,18 @@ export function createRecordedClient(context: Context): RecordedClient<PhoneNumb
   };
 }
 
+export function createRecordedOcClient(context: Context): RecordedClient<OperatorConnectClient> {
```

Comment:
I wouldn't abbreviate Operator Connect as OC, unless this is a widely known acronym.

Reply:
Refactored recorded client in a bit more OOP(ish)  style. That also solved naming issue.